.github/workflows/build_apps.yml

```diff
@@ -0,0 +1,41 @@
+name: Build_apps
+
+on:
+  pull_request:
+    branches: [master]
+  push:
+    branches: [master]
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, windows-latest]  # TODO cclauss: Add macos-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-python@v2
+      with:
+        python-version: 3.x
+    - name: Install dependencies for Ubuntu
+      if: matrix.os == 'ubuntu-latest'
+      run: sudo apt-get install libsdl2-2.0-0
```

Comment:
The `if` shouldn't be necessary. We shouldn't do a matrix build in the first place.

I would suggest to split this file up and run the build for Linux, macOS and Windows separately. Then you need no `if`s, because you know which OS this build job runs for.